pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -764,6 +764,11 @@ func (ctrl *Controller) cleanUpDuplicatedMC() error {
 		}
 		// delete the containerruntime mc if its degraded
 		if mc.Annotations[ctrlcommon.GeneratedByControllerVersionAnnotationKey] != version.Hash {
+			mc, err = ctrl.client.MachineconfigurationV1().MachineConfigs().Get(context.TODO(), mc.Name, metav1.GetOptions{})
```

Comment:
Hmm, the re-fetch here would overwrite the MC from before right? For now, I think we should just try printing it out

(Sorry for the confusion, when we discussed this, I meant to do this additional get as a separate PR to see if it makes a difference)